server/controllers/memoCtrl.js

```diff
@@ -139,6 +141,14 @@ function outdatedMemoData(offerings, startSelectionYear, memoData) {
   return true
 }
 
+function extendMemoWithStartDate(offerings, memoData) {
+  const offering = offerings.filter(o =>
+    memoData.applicationCodes.includes(o.round.applicationCodes[0].applicationCode)
+  )
+  const startDate = offering.length > 0 ? offering[0].round.firstTuitionDate : ''
```

Comment:
`offering[0].round.firstTuitionDate` it is to unsafe to extract such a long line it has to be more safe destructure